src/main/java/de/dennisguse/opentracks/AbstractActivity.java

```diff
@@ -36,13 +38,31 @@ protected void onCreate(Bundle savedInstanceState) {
         setVolumeControlStream(TTSManager.AUDIO_STREAM);
 
         setContentView(getRootView());
+        this.ThemeUpdate();
     }
 
+   public void ThemeUpdate() {
```

Comment:
Method name should be lower case and it should be stating what it is doing like `setThemeOled()`.